### DISCUSSION TITLE
Save Output as netCDF (.nc) file with parcels version 2.4.2 or more

### DISCUSSION THREAD

**Discussion by BenOwusu084** [Parcels-code/Parcels#1518] [Q&A]
Hi I am working with the version 2.4.2 of parcels and I have issues saving output file as a netCDF file. It throws an error that tells that the output file cannot be written as a .nc in this version of parcels but as a .zarr file. It works well when I save it as a .zarr file however I would like to save the output file as a .nc file instead of a zarr file. I know I have an option to downgrade the version of parcels to 2.0.0. Could someone please tell me how I can downgrade to a lower version which allows output file to be saved as .nc or if there's a way around it with the upgraded version 2.4.2 which I am using now. I would really appreciate this. Thanks

**Comment by erikvansebille** [comment]
Hi @BenOwusu084, you can easily change the zarr file to netcdf using something like below 
```python
import array as xr
ds = xr.open_zarr("FILENAME.zarr")
ds.to_netcdf("FILENAME.nc")
```
See also the instructions at https://docs.oceanparcels.org/en/latest/examples/tutorial_output.html#Reading-the-output-file

**Reply by BenOwusu084** [reply]
Thanks a lot @erikvansebille
